test/MembershipListenerTest.js

```diff
@@ -14,11 +14,13 @@
  * limitations under the License.
  */
 
+
 var HazelcastClient = require('../.').Client;
 var Controller = require('./RC');
 var expect = require('chai').expect;
 var DeferredPromise = require('../lib/Util').DeferredPromise;
 var MemberAttributeOperationType = require('../.').MemberAttributeOperationType;
+var MemberEvent = require('../lib/invocation/ClusterService').MemberEvent;
```

Comment:
`MemberEvent` must be public API. (exported in index.ts)

Reply:
I don't think it should be in index because user never uses it. We just return it to user.